util/cron/nightly

```diff
@@ -297,6 +297,25 @@ if (! (-r $logdir and -w $logdir and -d $logdir)) {
     print "Error: CHPL_NIGHTLY_LOGDIR ($logdir) not accessible\n";
 }
 
+$memleaksdir = $ENV{'CHPL_NIGHTLY_MEMLEAKS_DIR'};
+
+if ($memleaks ne "") {
+    # Default to known file share.
+    if ($memleaksdir eq "") {
+        $memleaksdir = "/data/sea/chapel/NightlyMemLeaks";
```

Comment:
I don't like hard coding the paths, but we do it several other places in nightly so it at least feels consistent.

I don't think we want the memleaks files to end up in .../Nightly, which is why I chose this default. But it only really makes sense in the -cron mode. I'll add a branch for -debug mode that uses $logdir, which is based on CHPL_NIGHTLY_LOGDIR.